### ISSUE TITLE
Try Typo CI?

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tskit#1482]
Is it worth trying out some CI to flag typos? I saw [this service](https://typoci.com) recently, might be worth a look.

Could well be more trouble that it's worth, but I thought I'd flag it as a possibilty.

**Comment by petrelharp** [comment]
I vote 'no' - we don't want to add more friction to the process. And, humans will need to read the text anyhow to make sure it makes sense; flagging a few typos doesn't seem worth it (even if it doesn't throw false positives...).

**Comment by jeromekelleher** [comment]
Yep, agreed. Let's see if others chime in and close in a few days if not.

**Comment by benjeffery** [comment]
We've had quite a few typos, a few get fixed in every (2-5)th PR. I think it might be worth adding to the release CI flow, at the point we push to test PyPI. That way we check occasionally but it doesn't add friction.

**Comment by grahamgower** [comment]
There must be a typo in the typoci.com site, because I was unable to find the app using the link to the marketplace (or any further details at all).

**Comment by benjeffery** [comment]
Whoops: https://mikerogers.io/2021/05/23/saying-goodbye-to-typo-ci

**Comment by jeromekelleher** [comment]
I don't know, seems like a lot of work on reflection now. Look at it this way - a few typos is actually a good thing because it gives interested users an easy way in contributing? So, if we got rid of them all, we'd be restricting our contributor pipeline :wink:

**Comment by molpopgen** [comment]
Like @petrelharp, I'm not in favor of this, for the same reasons.

**Comment by jeromekelleher** [comment]
I'm going to close this as there's more minus votes than positive.
